README.md

```diff
@@ -52,16 +53,14 @@ conn, err := gorqlite.Open("https://mary:secret2@server1.example.com:4001/?level
 conn, err := gorliqte.Open("https://server2.example.com:4001/?level=weak")
 // different port, setting the rqlite consistency level and timeout
 conn, err := gorqlite.Open("https://localhost:2265/?level=strong&timeout=30")
+// different port, setting the rqlite consistency level, timeout, and disabling cluster discovery
+conn, err := gorqlite.Open("https://localhost:2265/?level=strong&timeout=30&disableClusterDiscovery=true")
```

Comment:
So the only thing about this is that it implies that `isableClusterDiscovery` is something that `rqlited` supports, which of course it doesn't. I understand why you did it -- and if this is the only way, that's what we will have to do -- but it could confuse people.

Instead, is it possible to add a new method to the `conn` object, something like `DisableClusterDiscovery(bool)`?

Reply:
Good point. Yes, I can do that.

Reply:
Actually, I take that back. Parameters in the connection string shouldn't necessarily map to rqlite API options. Even `timeout` is not something that rqlited supports but is something is used to configure the client to time out after a certain period of time. Does that make sense?

Also, the cluster discovery happens when the connection is initialized, so it's going to be tricky to revert its changes later with something like `conn.DisableClusterDiscovery`.

Reply:
Yeah, your last point is the one I was afraid of (but just don't know the code well enough without looking at it).

Reply:
We could make it a second parameter to the `Open` function, but unfortunately that would be a breaking change, and I'd like to avoid that.

Reply:
Yeah, agreed. This client library, even though it has plenty of rough edges, has been quite useful to folks and we owe it a lot. I think it needs to be rewritten (or a new one added to the repo which will be more inline with how rqlite works) but until that happens, we shouldn't break the API.